assets/js/frontend/components/load-more-button/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+export const LoadMoreButton = ( { onClick } ) => (
+	<button
+		className="wc-block-reviews-by-product__load-more"
```

Comment:
These classnames are not generic enough. Given what this component does, does this need to be it's own component?

Reply:
> These classnames are not generic enough.

Absolutely, forgot to change the class names and move styles. :man_facepalming: 

Should be fixed in fde9e972de5bba44d64fe736d72f15705bb0de47.

> Given what this component does, does this need to be it's own component?

Considering we have to use it in several blocks, I think it makes sense creating it as a component that can be easily imported, right? If some day we want to change the class name, the text or add some attribute it will be easier if it's in a shared component than copy-&-pasted to several files.